src/components/FastServices.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "styled-components";
+
+const FastServices = () => {
+  return <h1>FastServices</h1>;
+};
+
+const Wrapper = styled.main`
```

Comment:
You have a lot of unused variables in your code. Use them, export them or delete them. Also, add noUnusedLocals option to the compiler settings.

Reply:
Yes, it just stolen CSS code for time being.. I will sort it in next steps.

Reply:
If this is not needed in this pr, then don't add this. Right now there is no option to test this css so I won't approve it.